students/ArchWu/Session3/list_lab.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+
+def series1(fruits):
+    answer = input("Please pick a fruit to add > ")
+    fruits.append(answer)
+    print(fruits)
+    answer = input("Please pick an integer > ")
+    while not answer.isdigit() or int(answer) > len(fruits):
+        answer = input("Please pick an integer > ")
+    print(fruits[int(answer) - 1])
+    answer = input("Please pick a fruit to add > ")
+    new_fruit = [answer]
+    fruits = new_fruit + fruits
+    print(fruits)
+    answer = input("Please pick a fruit to add > ")
+    fruits.insert(0, answer)
+    print(fruits)
+    for f in fruits:
+        if f[0] == 'P': print(f)
```

Comment:
One extra case you could consider is if the user enters a fruit with a lowercase p